### DISCUSSION TITLE
Method to emit events only to immediate parent

### DISCUSSION THREAD

**Discussion by viaativaweb** [livewire/livewire#2858] [3. Feature ideas]
When using a component with multiple instances and several listeners all over the place (eg: a custom text input component) blubbing events globally becomes a nightmare. More often than not we want a component event to be only used by the view that calls it, like it happens with Vue:

`$this.$emit('send', this.$refs.input.value)`
`<my-input v-on:send="saveMessage"></my-input>`

A `$emitParent('eventName')` method would solve this problem beautifully and save us from hacky solutions like sending custom event names to the component.

**Comment by ju5t** [comment]
This already exists: https://laravel-livewire.com/docs/2.x/events#scope-to-parents. You can even emit to specific components.
